tox.ini

```diff
@@ -17,6 +17,7 @@ passenv =
     aws_access_key_id
     aws_secret_access_key
     mturk_worker_id
+    threads
```

Comment:
This is to make sure that threads=1 in .travis.yml makes it into the tox environment so that we avoid running a whole bunch of gunicorn workers on Travis (since it defaults to scaling proportional to the number of CPUs)